crates/firewheel-core/Cargo.toml

```diff
@@ -17,9 +17,14 @@ all-features = true
 [features]
 symphonium = ["dep:symphonium"]
 resampler = ["symphonium?/resampler"]
+bevy = ["dep:bevy_ecs"]
 
 [dependencies]
+firewheel-macros.workspace = true
 arrayvec.workspace = true
 bitflags.workspace = true
 thunderdome.workspace = true
 symphonium = { version = "0.3", default-features = false, optional = true }
+smallvec.workspace = true
+glam = "0.29"
```

Comment:
Same for glam, it should be an optional dependency.

Reply:
Oh wait, I guess we're using those types internally for the spatial node. Nevermind.